docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,51 +1,134 @@
+the changes are available upstream in the OpenTelemetry repositories.
+* The distro preconfigures the collection of tracing and metrics signals, applying some opinionated defaults,
+such as which sources are collected by default.
+
+// Why you wouldn't choose this method
+// Still in tech preview?
+
+Get started with an Elastic Distribution for OpenTelemetry Language SDK:
+
+* preview:["The Elastic Distribution for OpenTelemetry Java is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-java[*Elastic Distribution for OpenTelemetry Java →*]
+* preview:["The Elastic Distribution for OpenTelemetry .NET is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-dotnet[*Elastic Distribution for OpenTelemetry .NET →*]
+* preview:["The Elastic Distribution for OpenTelemetry Node.js is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-node[*Elastic Distribution for OpenTelemetry Node.js →*]
+
+[NOTE]
+====
+For more details about OpenTelemetry distributions in general,
+visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
+====
+
+[float]
+[[apm-otel-api-sdk-elastic-agent]]
+== OpenTelemetry API/SDK + Elastic APM agent
+
+Use the OpenTelemetry API/SDKs with Elastic APM agents to translate OpenTelemetry API calls to Elastic APM API calls.
+
+image::images/apm-otel-api-sdk-elastic-agent.png[]
 
-<<apm-instrument-apps-otel,Get started>>.
+// Why you _would_ choose this method
+This allows you to reuse your existing OpenTelemetry instrumentation to create Elastic APM transactions and spans--avoiding
+vendor lock-in and having to redo manual instrumentation.
 
-**OpenTelemetry collector**
+// Why you would _not_ choose this method
+However, not all features of the OpenTelemetry API are supported when using this approach, and
+not all Elastic APM agents support this approach.
+
+Find more details about how to use an OpenTelemetry API or SDK with an Elastic APM agent and
+which OpenTelemetry API features are supported in the APM agent documentation:
+
+* {apm-java-ref}/opentelemetry-bridge.html[*APM Java agent →*]
+* {apm-dotnet-ref}/opentelemetry-bridge.html[*APM .NET agent →*]
+* {apm-node-ref}/opentelemetry-bridge.html[*APM Node.js agent →*]
+* {apm-py-ref}/opentelemetry-bridge.html[*APM Python agent →*]
+
+// <<apm-open-telemetry-with-elastic,*Get started with an OpenTelemetry API/SDK + an Elastic APM agent →*>>
+
+[float]
+[[apm-otel-native-support]]
+== OpenTelemetry native support
```

Comment:
With that title, this section somehow subsumes two of the other sections (because there we also go through the OTel native support):

- Elastic Distributions for OpenTelemetry Language SDKs
- AWS Lambda Collector exporter

The main point here that we want to make is that users can also use any upstream / custom OTel components (SDKs or collector) to send data to Elastic through OTLP.

Let's maybe change the title to something along those lines:
- `OpenTelemetry-owned Language SDKs and Collector`
- `Vanilla OpenTelemetry Language SDKs and Collector`
- `Upstream OpenTelemetry Language SDKs and Collector`
- `Vanilla OpenTelemetry Components`
- ...